README.md

```
@@ -4,91 +4,442 @@
+- <p id="Monitor">Install Helm Chart</p>
+```bash
+curl -fsSL -o get_helm.sh https://raw.githubusercontent.com/helm/helm/main/scripts/get-helm-3
+```
+```bash
+chmod 700 get_helm.sh
+```
+```bash
+./get_helm.sh
+```
+
+#
```

Markdownlint (0.37.0)

Multiple headings with the same content

(MD024, no-duplicate-heading)